src/pkg/services/m365/api/mail_test.go

```diff
@@ -370,34 +372,175 @@ func (suite *MailAPIIntgSuite) TestMail_attachmentListDownload() {
 	}
 }
 
-func (suite *MailAPIIntgSuite) TestMail_PostLargeAttachment() {
```

Comment:
I've replaced this test with a new one which fetches attachments after posting them.